src/main/java/bio/terra/service/dataset/DatasetBucketDao.java

```diff
@@ -99,6 +105,12 @@ boolean datasetBucketLinkExists(UUID datasetId, UUID bucketResourceId) {
         return (count == 1);
     }
 
+    public List<UUID> getBucketForDatasetId(UUID datasetId) {
```

Comment:
Caught up offline and it makes sense. I think maybe just renaming this method to something like `getBucketResourceIdForDatasetId` might be a little less confusing